docker/beta/follow_dispatcher/consul-template-follow-dispatcher.conf

```diff
@@ -0,0 +1,13 @@
+template {
+    source = "/code/listenbrainz/consul_config.py.ctmpl"
+    destination = "/code/listenbrainz/listenbrainz/config.py"
+}
+
+exec {
+    # TODO: Working directory
+    command = "python3 manage.py run_follow_server -h 0.0.0.0 -p 3031"
```

Comment:
To run in a working directory, use cd && python, but consider consul-template documentation about shells and signals: https://github.com/hashicorp/consul-template#exec-mode

It's possible that we're going to need a separate shell script in /usr/local/bin that does cd and then uses exec